test/box-py/args.test.py

```diff
@@ -59,4 +59,10 @@
                    script + \
                    " 1 2 3 --help")
 
+server.test_option("-b")
+server.test_option("-bl -e ''")
+
+server.test_option("-jon -e ''")
+server.test_option("-j on -e ''")
```

Comment:
The checkpatch warning is about a trailing whitespace:

```diff
ERROR:TRAILING_NEWLINE: trailing newline
#444: FILE: test/box-py/args.result:182:
+tarantool -j on -e 
+
```

At least you can do something like the following:

```python
cmd = <...>
res = server.test_option_get(cmd, silent=True)
print(cmd)
print(res)
```

It uses [test_option_get](https://github.com/tarantool/test-run/blob/07a0c00c8b54e1e4bc2a458ce7e76b5cd790eefb/lib/tarantool_server.py#L1198) from test-run.

I would rewrite the test to luatest and drop `test_option` and `test_option_get` from test-run, but not necessarily in this patchset.